wesop/settings.py

```diff
@@ -0,0 +1,145 @@
+"""
+Django settings for wesop project.
+
+Generated by 'django-admin startproject' using Django 4.0.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+
+from pathlib     import Path
+from my_settings import *
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-6*60-_-6gh0m!t-=z#i4bl#xjtf^a^0wo8u75z52g0sxhapu%b'
```

Comment:
SECRET_KEY도 my_settings에 숨겨 주세요.
```suggestion
SECRET_KEY = SECRET_KEY
```